docs/_posts/2014-08-07-databus.markdown

```diff
@@ -58,6 +58,7 @@ Client jerseyClient = new JerseyClientBuilder(metricRegistry)
                 .using(Executors.newSingleThreadExecutor())
                 .using(Jackson.newObjectMapper())
                 .build("DatabusClient");
```

Comment:
here and below consider to chain calls
```suggestion
                .build("DatabusClient")
                .register(JacksonJasonProvider.class);
```

Reply:
not required anymore, since replaced with client register validation